source/Calamari.Shared/Integration/Proxies/IProxySettings.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Octopus.CoreUtilities;
+
+namespace Calamari.Integration.Proxies
+{
+    public interface IProxySettings
```

Comment:
We should take more care in preserving file history when renaming the file (minimise add/deletes), it makes annotating harder in the future.